tests/index.js

```diff
@@ -170,35 +170,57 @@ test('sagaTestEngine throws under bad conditions', t => {
     () => sagaTestEngine(f, badMapping),
     'Env Mapping is missing a value for "key"')
 
+  // Bad mapping for saga that yields obj with anonymous function
+  function namedFunction() {}
+  const f2 = function*() {
+    yield { func: namedFunction }
+  }
+  const badMapping2 = [['no', 'functions']]
+  t.throws(
+    () => sagaTestEngine(f2, badMapping2),
+    `Env Mapping is missing a value for ${JSON.stringify({ "func": "[Function: namedFunction]" }, null, 2)}`
+  )
+
+  // Bad mapping for saga that yields object with anonymous function
+  var anonymousFunction = function() { return 'something'; }
+  const f3 = function*() {
+    yield { func: anonymousFunction }
+  }
+  const badMapping3 = [['also no', 'functions']]
+  t.throws(
+    () => sagaTestEngine(f3, badMapping3),
+    `Env Mapping is missing a value for ${JSON.stringify({ func: `[Function]: ${anonymousFunction.toString()}` }, null, 2)}`
+  )
```

Comment:
I think you're right that we can just remove or water down that particular test. The point is to provide as much information as possible to the user that the node engine allows.